### PR TITLE
Add system info utility

### PR DESCRIPTION
**Pull request by awharn**
Adds a utility to pass back an object with the architecture and platform of the system.
  - Should help with determining what version of the Zowe Daemon binary to install

### CODE REVIEW

File: packages/utilities/__tests__/ProcessUtils.test.ts [hunk 1]
```diff
@@ -102,4 +102,120 @@ describe("ProcessUtils tests", () => {
...
     });
 
+    describe("getBasicSystemInfo", () => {
+        let realPlatform: string;
+        let realArch: string;
+
+        beforeAll(() => {
+            realPlatform = process.platform;
+            realArch = process.arch;
+        })
```
Comment (zFernand0) [inline packages/utilities/__tests__/ProcessUtils.test.ts:112]: tiny lint warning

**Review by zFernand0 — COMMENTED** [review]
PR LGTM!
Should this be added to master/zowe-v1-lts ?

**Comment by awharn** [comment]
> PR LGTM! Should this be added to master/zowe-v1-lts ?

I don't know that it needs to be, but we can - this is kind of intended just for helping to determine basic OS info so that we can attempt to install the daemon. I really don't like messing with `process`, so I thought a little helper like this in Imperative would be a good way to go, and we can expand upon it in the future if we need to.

File: packages/utilities/__tests__/ProcessUtils.test.ts (hunk 1 above)
Comment (awharn) [reply packages/utilities/__tests__/ProcessUtils.test.ts:112]: Thanks.

**Review by t1m0thyj — APPROVED** [review]

**Review by zFernand0 — APPROVED** [review]
LGTM! : )

**Pull request merged**